src/check_sentryio_limits.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""InnoGames Monitoring Plugins - sentry.io event limit check
+
+The script will exit with:
+ - 0 (OK) All the limits provided to this check are kept
+ - 1 (WARNING) Limits are either set to high or not at all
```

Comment:
Let's not send some poor sysadmin debugging the script when there is network foo or the authentication failed.

I added to check for HTTP status code 200 further below.

```suggestion
 - 1 (WARNING) Limits are either set to high or not at all
 - 3 (UNKNOWN) Error while fetching team or project info via HTTP API
```